### PR TITLE
Add HoangVu personal page

### PR DESCRIPTION
**Pull request by JeFlame**
## Description

This PR is to add my personal page (HoangVu)

## Additional Information

- [x] I have read the [contributing docs](/scaffold-eth/scaffold-eth-2/blob/main/CONTRIBUTING.md) (if this is your first contribution)
- [x] This is not a duplicate of any [existing pull request](https://github.com/scaffold-eth/scaffold-eth-2/pulls)

## Related Issues

Your ENS/address: 0xDbD3834299D363B4Fb84Aef9309d3068E52CFc0C

### CODE REVIEW

**Comment by JeFlame** [comment]
@carletex Please help me review this PR

File: packages/nextjs/app/builders/0xDbD3834299D363B4Fb84Aef9309d3068E52CFc0C/page.tsx [hunk 1]
```diff
@@ -0,0 +1,94 @@
...
+              </div>
+            </div>
+
+            <div className="flex justify-center items-center gap-8 flex-col sm:flex-row">
+              <div className="flex flex-col py-6 text-center items-center max-w-xs rounded-3xl">
+                <div className="flex flex-col bg-base-100 px-6 py-6 text-center items-center max-w-xs rounded-3xl mb-6">
+                  <AcademicCapIcon className="h-8 w-8 fill-secondary" />
+                  <p>
+                    Explore my projects on{" "}
+                    <Link href="https://github.com/JeFlame" passHref className="link" target="_blank">
```
Comment (technophile-04) [inline packages/nextjs/app/builders/0xDbD3834299D363B4Fb84Aef9309d3068E52CFc0C/page.tsx:43]: It is ideally prefered and good pattern to follow that you use `a` tag for extenal links. 

Link component makes sense  and are useful for client side navigation b/w internal routes, but yeah for external link people use `a` tag

**Comment by technophile-04** [comment]
Looks clean and responsive J !! Just added a small comment, could you please update it ?

File: packages/nextjs/app/builders/0xDbD3834299D363B4Fb84Aef9309d3068E52CFc0C/page.tsx (hunk 1 above)
Comment (JeFlame) [reply packages/nextjs/app/builders/0xDbD3834299D363B4Fb84Aef9309d3068E52CFc0C/page.tsx]: Thank u, I'll fix it

**Comment by JeFlame** [comment]
> Looks clean and responsive J !! Just added a small comment, could you please update it ?
I updated the code, please help me review my PR again

**Review by technophile-04 — APPROVED** [review]
Thanks @JeFlame !!

**Pull request merged**